frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -93,18 +92,15 @@ export function DashboardInfoSidebar({
       </ContentSection>
 
       <ContentSection>
-        <FormField
-          title={t`Auto-apply filters`}
-          orientation="horizontal"
-          htmlFor={autoApplyFilterToggleId}
-        >
-          <Toggle
-            disabled={!canWrite}
-            id={autoApplyFilterToggleId}
-            value={dashboard.auto_apply_filters}
-            onChange={handleToggleAutoApplyFilters}
-          />
-        </FormField>
+        <SpaceBetweenSwitch
+          disabled={!canWrite}
+          label={t`Auto-apply filters`}
+          labelPosition="left"
+          size="sm"
```

Comment:
Do we want this to be `sm` or `md`? `sm` looks a bit small compared to the old one.

Reply:
the label size also scales with the size, `md` was very large for the label

Reply:
I ran into a data migration issue trying to spin up the branch but if the `sm` size is what's in the screenshots that looks ok to me. It **is** expected that there's a different between the old stuff and the new stuff since the Mantine based components are using an updated type system. The different isn't supposed to be crazy though. 

Out of curiosity what's the "default" size for Switch? Ideally the default one should be kicking out something closer to 14px "normal" type I would hope? If we have to set `sm` on all switches just to get the base case, I think something screwy is going on.